### DISCUSSION TITLE
Allow larger button icon sizes

### DISCUSSION THREAD

**Discussion by bernhardh** [livewire/flux#547] [Feature Requests]
There is no way to change the icon size of a button (or `sidebar.toggle` component) to larger sizes then `xs` and `sm`. This is way to small for a lot of usecases and expecially for accessibility / WAI, disabled people.

There a a lot of example, but especially icon-only buttons are very tiny to click. The buttons are too small regarding all common `Touch target size` definition:

![Image](https://github.com/user-attachments/assets/7521961e-a593-4572-9d08-cecb60aa6f80)

This works

```html
<flux:button icon-trailing="home" size="xs"></flux:button>
<flux:button icon-trailing="home" size="sm"></flux:button>
```

this not:

```htm
<flux:button icon-trailing="home" size="md"></flux:button>
<flux:button icon-trailing="home" size="lg"></flux:button>
...
```

Plus: It would be great, if there would be responsive options, definiing button sizes depending on the screen size.

**Comment by LasseRafn** [comment]
doesn't solve your issue entirely but you can do icon-variant="outline" to get a slightly larger icon, and then add classes on the button to change the size, such as

```
<flux:button icon-trailing="home" size="sm" class="w-32 h-32 lg:w-16 lg:h-16"></flux:button>
```

again, doesn't solve the icon problem, but its a start I guess.

**Comment by calebporzio** [comment]
Thanks for the feature request, we'll keep this in mind.
